### PR TITLE
AC-214 - Apply MVP - PatientListActivity

### PR DESCRIPTION
**Pull request by tmarzeion**
This one works except one new function (starting visit when clicking patient with no active visit).
This is caused by retrofit request, i will fix this up soon

//edit: since start visit function isn't implemented to actual upstream code, I think we can fix it later, after MVP apply.

### CODE REVIEW

**Comment by AdamGrzybkowski** [comment]
@tmarzeion Please, resolve conflicts :)

**Comment by tmarzeion** [comment]
@AdamGrzybkowski Solved :)

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/formentrypatientlist/FormEntryPatientListActivity.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+        }
+
+        // Create fragment
+        FormEntryPatientListFragment formEntryPatientListFragment =
+                (FormEntryPatientListFragment) getSupportFragmentManager().findFragmentById(R.id.formEntryPatientListContentFrame);
+        if (formEntryPatientListFragment == null) {
+            formEntryPatientListFragment = FormEntryPatientListFragment.newInstance();
+        }
+        addFragmentToActivity(getSupportFragmentManager(),
```
Comment (AdamGrzybkowski) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/formentrypatientlist/FormEntryPatientListActivity.java]: @tmarzeion check if fragment is not already added

Comment (tmarzeion) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/formentrypatientlist/FormEntryPatientListActivity.java]: Fixed

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/formentrypatientlist/FormEntryPatientListFragment.java [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+import org.openmrs.mobile.utilities.StringUtils;
+
+import java.util.List;
+
+public class FormEntryPatientListFragment extends ACBaseFragment implements  FormEntryPatientListContract.View {
+
+    FormEntryPatientListContract.Presenter mPresenter;
+    private RecyclerView mPatientRecyclerView;
+    public String mSelectedPatientUUID;
+    public Long mSelectedPatientID;
```
Comment (tmarzeion) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/formentrypatientlist/FormEntryPatientListFragment.java]: This one is some old code logic leftover, its not needed anymore.
Fixed

**Pull request merged**